erpnext/patches/v12_0/fix_quotation_expired_status.py

```diff
@@ -0,0 +1,37 @@
+import frappe
+
+
+def execute():
+    # fixes status of quotations which have status 'Expired' despite having valid sales order created
+
+	# filter out submitted expired quotations which has sales order created
+	cond = "qo.docstatus = 1 and qo.status = 'Expired'"
+	invalid_so_against_quo = """
```

Comment:
invalid_so_against_quo has never used in below query, you have used so_against_quo